ops/terraform/modules/resources/cw_metric_filters/variables.tf

```diff
@@ -0,0 +1,8 @@
+variable "env" {
+  type        = string
+}
+
+variable "log_groups" {
+  type    = object({access: string})
+  default = null
+}
```

Comment:
I don't think `log_groups` is used anywhere? I'd remove it.

(You have it as a `local`, but I don't think that requires also declaring it as a `variable`.)